hasher/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+pub mod bytes;
+pub mod errors;
+pub mod keccak;
+pub mod poseidon;
+pub mod sha256;
+pub mod syscalls;
+pub mod zero_bytes;
+pub mod zero_indexed_leaf;
```

Comment:
same for hasher it's actively in development, I would like to put libraries here which are being phased out or a really very stable.
One thing I want to change in the hasher trait is move the discriminator trait to light-utils and possibly move the data hasher trait to the sdk as well.